### ISSUE TITLE
CTGP Nitro crashes with 4MB memory

### ISSUE THREAD

**Issue by spellboundtriangle** [DS-Homebrew/nds-bootstrap#1379]
**Your console:** DSi

**Launch Method:** DSONE flashcard, locked or unlocked SCFG

**Version used:** Nightly aa78368

**Game tested/used/applicable:** [CTGP Nitro 1.0.0](https://dswiki.garhoogin.com/ctgp-n/)

**Expected Behavior:** The game would start and play normally

**Actual Behavior:** The game crashes with a red crash handler screen after the Nintendo logo
![IMG_20220315_183401052](https://user-images.githubusercontent.com/79382000/158489481-304205c6-a52e-49f3-99ba-2841b21984fa.jpg)

### Steps to reproduce
1. Start CTGP Nitro 1.0.0 in B4DS mode, 4MB memory
2. Wait until after the Nintendo logo

### nds-bootstrap.ini
```
[NDS-BOOTSTRAP]
USE_ROM_REGION = 1
NDS_PATH = fat:/roms/nds/CTGP Nitro.nds
SAV_PATH = fat:/roms/nds/saves/CTGP Nitro.sav
RAM_DRIVE_PATH = sd:/null.img
GUI_LANGUAGE = en
LANGUAGE = -1
REGION = 1
DSI_MODE = 1
EXTENDED_MEMORY = 0
DONOR_SDK_VER = 0
PATCH_MPU_REGION = 0
PATCH_MPU_SIZE = 0
FORCE_SLEEP_PATCH = 0
BOOST_CPU = 0
BOOST_VRAM = 0
CARD_READ_DMA = 1
ASYNC_CARD_READ = 0
LOGGING = 1
DEBUG = 1
```

### Logs
[NDSBTSRP.LOG](https://github.com/DS-Homebrew/nds-bootstrap/files/8257886/NDSBTSRP.LOG)

### Other notes
- This does not occur with version CTGP Nitro 0.0.1, if I recall correctly
- This does not occur with 8MB+ mode (or a MEP, assuming 8MB+ mode is accurate to it)
- As far as I know, this should _not_ simply be a general MEP requirement, as from what I have heard, a MEP is only used for storing the ROM in, if it can fit (which it does not)

**Comment by spellboundtriangle** [comment]
This appears to be fixed as of cf780bb

**Comment by RocketRobz** [comment]
43965609 is the actual commit which fixes this issue.
